pypac/api.py

```diff
@@ -35,7 +36,9 @@ def get_pac(url=None, js=None, from_os_settings=True, from_dns=True, timeout=2,
     :raises MalformedPacError: If something that claims to be a PAC file was obtained but could not be parsed.
     """
     if url:
-        downloaded_pac = download_pac([url], timeout=timeout, allowed_content_types=allowed_content_types)
+        downloaded_pac = download_pac([url], timeout=timeout,
+                                      allowed_content_types=allowed_content_types,
+                                      session=session)

```

Comment:
Please update the other call to `download_pac()` on line 71.

Reply:
Sorry, but I don't understand what you mean :(